test/fixtures/mson-inner-inheritance.json

```diff
@@ -20,33 +20,76 @@
               "content": {
                 "key": {
                   "element": "string",
-                  "content": "A"
+                  "content": "id"
                 },
                 "value": {
-                  "element": "object",
+                  "element": "number"
+                }
+              }
+            }
+          ]
+        },
+        {
+          "element": "object",
+          "meta": {
+            "id": "Person"
+          },
+          "content": [
+            {
+              "element": "member",
+              "content": {
+                "key": {
+                  "element": "string",
+                  "content": "user"
+                },
+                "value": {
+                  "element": "extend",
                   "content": [
                     {
-                      "element": "member",
-                      "content": {
-                        "key": {
-                          "element": "string",
-                          "content": "a"
-                        },
-                        "value": {
-                          "element": "string"
+                      "element": "object",
+                      "meta": {
+                        "ref": "User"
+                      },
+                      "content": [
+                        {
+                          "element": "member",
+                          "content": {
+                            "key": {
+                              "element": "string",
+                              "content": "id"
+                            },
+                            "value": {
+                              "element": "number"
+                            }
+                          }
                         }
-                      }
+                      ]
+                    },
+                    {
+                      "element": "object"
```

Comment:
Question: are you all OK with using this empty object like this? Is there a need to include it like this?